include/z64player.h

```diff
@@ -459,8 +459,16 @@ typedef enum {
     /* 0x67 */ PLAYER_CSMODE_MAX
 } PlayerCutsceneMode;
 
+typedef enum {
+    /* 0 */ PLAYER_LEDGE_CLIMB_NONE,
+    /* 1 */ PLAYER_LEDGE_CLIMB_1,
+    /* 2 */ PLAYER_LEDGE_CLIMB_2,
+    /* 3 */ PLAYER_LEDGE_CLIMB_3,
+    /* 4 */ PLAYER_LEDGE_CLIMB_4
```

Comment:
I never realized there were different types of ledge climbing. Do you have a rough sense as to what the difference between these are?

Reply:
hop, small jump, large jump etc